week-1-ts-ecosystem/w01e01-typescript-basics/src/logic.ts

```diff
@@ -0,0 +1,11 @@
+import {User} from './data'
+
+function hasAddress(user: User): boolean {
+    return Boolean(user.address)
+  }
+  
+function hasGivenAge(requiredAge: number): Function {
```

Comment:
typ `Function` jest zbyt ogólny, każda funkcja to `Function`. Możesz tutaj nie pisać żadnego typu - wtedy zdać sie na wnioskowanie z Typescipt. On będzie znał dokładny kształt danych wtedy tej funkcji - jej parametry i zwracany typ.